code/modules/antagonists/changeling/changeling.dm

```diff
@@ -934,6 +937,7 @@
 	var/laugh_type
 	var/age
 	var/list/quirks = list()
+	var/gun_permit
```

Comment:
So, I know I haven't reviewed the previous ones properly, but I'd like for you to go through those variables and add some documentation that just explains what those variables are.
You can do variable documentation like this:
```
/// A variable that's used to calculate the distance between our mob and the sun.
var/distance_from_sun
```

Reply:
Went through and commented on everything as best as I could. Let me know if I missed something or if it's not enough!